amundsen_application/static/css/_variables-default.scss

```diff
@@ -14,7 +14,7 @@ $brand-primary: $brand-color-4 !default;
 
 /* Scaffolding */
 $body-bg:           $white !default;
-$body-bg-secondary: $gray5 !default;
+$body-bg-secondary: #fcfcff !default;
```

Comment:
hardcoded??

Reply:
Oh yeah, this should be $gray0. I'll fix this in the other PR and revert this.